hk0i/task/add rakefile support

hk0i opened the pull request:
<!--
    Check those that apply, and delete the ones that don't
-->

* [x] I updated the CHANGELOG.md
* [ ] I added an Option to enable / disable this feature
* [x] I tested the changes in this pull request myself

---

Let me know if I need to add any sort of tests (I'm not sure where or how I would do this)

Cheers! ;)

Closes #184

hk0i committed:
Add Rakefile support

Resolves refined-bitbucket/refined-bitbucket#184 - Rakefile support

  - Adds `Rakefile` as 'Universally named file' in `language-ext.js`

hk0i committed:
update changelog

- adds notes about rakefile support to changelog

reyronald commented on CHANGELOG.md:
```diff
@@ -8,8 +8,10 @@
-* **Vagrantfile & Jenkinsfile**: Support for Jenkinsfile & Vagrantfile syntax highlighting.
+* **Vagrantfile, Jenkinsfile, and Rakefile**: Support for Jenkinsfile, Vagrantfile and Rakefile syntax highlighting.
   [pull request #174](https://github.com/refined-bitbucket/refined-bitbucket/pull/174).
+  resolves [issue #184](https://github.com/refined-bitbucket/refined-bitbucket/issues/184).
```
Please include the link for the current pull request as well!

`[pull request #186](https://github.com/refined-bitbucket/refined-bitbucket/pull/186)`

hk0i replied:
Yeah I was actually wondering if there was a way to add this before actually opening the PR, is it possible? Of course I can add it now very easily, I'm mostly wondering for any future PRs.

reyronald replied:
Well I usually push the CHANGELOG change in a later commit after I've already opened the PR and that's how I know, but you could also check the most recently opened issue or PR and just add 1 to the number since ids are generated sequentially 😅 .

reyronald commented on CHANGELOG.md:
```diff
@@ -8,8 +8,10 @@
+* **Vagrantfile, Jenkinsfile, and Rakefile**: Support for Jenkinsfile, Vagrantfile and Rakefile syntax highlighting.
   [pull request #174](https://github.com/refined-bitbucket/refined-bitbucket/pull/174).
+  resolves [issue #184](https://github.com/refined-bitbucket/refined-bitbucket/issues/184).
+
```
Remove this empty line

reyronald commented on src/syntax-highlight/language-ext.js:
```diff
@@ -69,5 +69,6 @@ module.exports = {
     jenkinsfile: 'language-groovy',
-    vagrantfile: 'language-ruby'
+    vagrantfile: 'language-ruby',
+    rakefile: 'language-ruby'
```
I would like to keep them alphabetically ordered, could you put `rakefile` between `jenkinsfile` and `vagrantfile` please?

hk0i replied:
Yeah I totally agree, that makes sense.

reyronald reviewed (changes requested):
Hey @hk0i ! Thanks for your contribution!

Sorry it took me so long to review, I've been busy !

reyronald commented:
No tests are necessary for this! =) 

Requested some minor form changes, after that it'll be ready to merge!

hk0i commented:
FYI: I will likely get around to resolving these sometime later tonight/tomorrow.
Thanks for the feedback!

hk0i committed:
Implement minor code review changes

(hk0i/task/add-rakefile-support)

  - Sorts file names by name in `language-ext.js`
  - Adds this pull request to `CHANGELOG.md`
  - Removes blank line between bullet points in `CHANGELOG.md`
